### ISSUE TITLE
Outdated gem info: fnordmetric

### ISSUE THREAD

**Issue by vassilevsky** [rubytoolbox/rubytoolbox#171]
https://www.ruby-toolbox.com/projects/fnordmetric

When I try to open its github page, an entirely different gem opens.

I would like to help update the information, but I don't know how :) I've checked the catalog repo and I don't see a way to edit the info, other than delete this gem, which is not a desired solution :)

**Comment by colszowka** [comment]
Hey @vassilevsky - this is unfortunately unfixable on our end. Here's what happened:

* Author released a gem, pointing to https://github.com/paulasmuth/fnordmetric as one of it's urls, i.e. it's homepage url
* Author subsequently renamed the repository on github and put a completely different project in there, it seems

I cannot find the original source code elsewhere on the related github account either. Should we just drop this obviously abandoned project from the category?

**Comment by franklinyu** [comment]
Maybe @asmuth renamed his account and removed the repository.

**Comment by colszowka** [comment]
I don't think so, then GitHub would not automagically redirect to the other repo, they only do that for moved repos, for example try out https://GitHub.com/jnicklas/carrierwave

As an alternative, we might consider blacklisting specific repos from display in the toolbox

**Comment by FranklinYu** [comment]
I like the blacklist idea. We should also blacklist "Homepage" and "Source Code" link. "Documentation" should be fine, since it is generated by the gem itself and won't go away.

**Comment by colszowka** [comment]
@FranklinYu Excellent catch! #383 actually fixes a different problem, but we can use it to get this one here resolved as well - thanks for bringing it up! Would you like to send a PR that adds it to the list (similar to #384)? Otherwise I'll look into it in the next days
